src/wp-includes/class-wp-autoload.php

```diff
@@ -505,10 +505,10 @@ public static function autoload_core( $class_name ) {
 		 * to ensure that constants are defined.
 		 */
 		if ( str_starts_with( $class_name, 'simplepie' ) ) {
-			require_once ABSPATH . static::CLASSES_PATHS['simplepie'];
+			require ABSPATH . static::CLASSES_PATHS['simplepie'];
 		}
```

Comment:
This has failed unit tests because there is an early `return` missing for the SimplePie classes. Without that return, all classes from SimplePie match both conditions here and are required twice.